packages/playwright-test/types/test.d.ts

```diff
@@ -1268,6 +1268,15 @@ export interface FullConfig<TestArgs = {}, WorkerArgs = {}> {
    *
    */
   webServer: WebServerConfig | null;
+  /**
+   * > NOTE: This does not include test-level attachments. See
+   * [testInfo.attach(name[, options])](https://playwright.dev/docs/api/class-testinfo#test-info-attach) and
+   * [testInfo.attachments](https://playwright.dev/docs/api/class-testinfo#test-info-attachments) for working with test-level
+   * attachments.
+   *
+   * The list of files or buffers attached for the overall Playwright Test run. Some reporters show attachments.
+   */
+  attachments: { name: string, path?: string, body?: Buffer, contentType: string }[];
```

Comment:
fundamental question: we started with `testInfo.attachments` array initially, but IIRC we now consider using `testInfo.attach()` a better practice? Will the same happen here too? Shall we have a method right away?

Reply:
Mayb—but let's see if this gets used! I mentioned adding it yesterday, but we decided to punt on it for now.